### DISCUSSION TITLE
This plugin install doesnot work at all

### DISCUSSION THREAD

**Discussion by lasag-2511** [mattieserver/netbox-topology-views#310] [General]
### NetBox version

3.3.0

### Topology Views version

3.5.0

### Steps to Reproduce

python3 manage.py migrate netbox_topology_views
python3: can't open file 'manage.py': [Errno 2] No such file or directory

### Expected Behavior

python3 manage.py migrate netbox_topology_views
python3: can't open file 'manage.py': [Errno 2] No such file or directory

### Observed Behavior

python3 manage.py migrate netbox_topology_views
python3: can't open file 'manage.py': [Errno 2] No such file or directory

**Comment by dreng** [comment]
First of all, please put more effort in your bug report. You neither filled all fields in a meaningful way nor did you choose a proper title. Second, you reported a bug where it seems that not the plugin is the problem, but the fact that you did not follow the instructions correctly.

"'manage.py': [Errno 2] No such file or directory" is a pretty clear error message. You tried to use manage.py but you were not in the correct directory.

For the future, please try not to unload your frustration on us. We did not cause your problem. Moreover, we spend our free time here to provide a free plugin. We certainly don't deserve a rant, no matter how frustrated someone may be.

-> Converted bug report into discussion
